src/blocks/FoldableList/__stories__/FoldableList.mdx

```diff
@@ -23,7 +23,7 @@ import * as FoldableListStories from './FoldableList.stories.tsx';
 
 - `title: string` — Foldable list title (on the right)
 - `text: string` — Foldable item text
-- `link?: Link` — Link below the text (see [Link](?id=components-links-and-buttons-link--default&viewMode=docs))
+- `link?: Link` — Link below the text (see [Link](?id=components-links-and-buttons-link--default))
```

Comment:
It's not working right now.

must be https://preview.gravity-ui.com/page-constructor/1168/?path=/docs/documentation-types--docs#link